nixos/modules/services/web-apps/outline.nix

```
@@ -287,6 +287,43 @@ in
       );
     };
 
+    discordAuthentication = lib.mkOption {
+      description = ''
+        To configure Discord auth, you'll need to create an application at
+        https://discord.com/developers/applications/
+
+        See https://docs.getoutline.com/s/hosting/doc/discord-g4JdWFFub6
+        for details on setting up your Discord app.
+      '';
+      default = null;
+      type = lib.types.nullOr (
```

A bit unnecessary `null` imho
```suggestion
      default = { };
      type =

---

I'm unsure about changing this here, it is more semantically correct I guess, but it's different from the rest of the file. I feel like it's something more fit for another PR.